app/controllers/admin_controller.rb

```diff
@@ -13,10 +12,10 @@ def index
     @username = session[:cas_user]
     @email = session[:cas_extra_attributes][:mail]
     @last_name = session[:cas_extra_attributes][:sn]
-    if !session[:welcome]
-      flash[:success] = "Welcome #{@username}!  You have been sucessfully logged in!"
-      session[:welcome] = true;
-    end
+    return if session[:welcome]
+    flash[:success] = "Welcome #{@username}!  You have been sucessfully logged \
```

Comment:
I'm not worried about the line length here from rubo cop.  I think the slash is more confusing than helpful.